docs/usage/type-conversions.md

```diff
@@ -65,26 +67,33 @@ the same type as the original object, we proxy `tuple` and `bytes` objects.
 The following immutable types are implicitly converted from Javascript to
 Python:
 
-| Python          | Javascript          |
-|-----------------|---------------------|
-| `int`           | `Number`            |
-| `float`         | `Number`            |
-| `str`           | `String`            |
-| `bool`          | `Boolean`           |
-| `None`          | `undefined`         |
+| Python          | Javascript            |
+|-----------------|-----------------------|
+| `int`           | `Number` or `BigInt`* |
+| `float`         | `Number`              |
+| `str`           | `String`              |
+| `bool`          | `Boolean`             |
+| `None`          | `undefined`           |
+
+* An `int` is converted to a `Number` if the `int` is between -2^{53} and 2^{53}
+  inclusive, otherwise it is converted to a `BigInt`.
```

Comment:
Maybe add,
"if the browser supports BigInt" and indicate what would happen otherwise.

Reply:
Currently it converts it to a number. Maybe we should throw an error though in interest of the principle of least surprise.

Reply:
Since only 5% of browsers don't support bigint it's probably not worth worrying this case too much.